PairedEndSingleSampleWf-fc-b37.wdl

```diff
@@ -0,0 +1,1703 @@
+## Copyright Broad Institute, 2017
```

Comment:
This is enormous.  Did you just inline the existing subworkflows?

Reply:
This workflow was original repurposed by ruchim, I just updated to reflect any updates from dsde-pipelines/genomes_in_the_cloud/single_sample/PairedSingleSampleWf.wdl

Reply:
Is there a way to find out what your changes were?  This thing is enormous.

Reply:
A change to make the .alt reference optional for the SamToFastqAndBwaMemAndMba task. Everything else is the same.